### PR TITLE
Fixup namespaces

### PR DESCRIPTION
**Pull request by pranavkm**

### CODE REVIEW

**Comment by pranavkm** [comment]
cc @hishamco

**Comment by hishamco** [comment]
There are some tests are failed!!
Shall I close my PR #399?

File: test/Microsoft.Extensions.Localization.Tests/ResourceManagerStringLocalizerTest.cs [hunk 1]
```diff
@@ -229,13 +229,7 @@ private static int GetCultureInfoDepth(CultureInfo culture)
...
         private TestSink Sink { get; } = new TestSink();
 
-        private ILogger Logger
-        {
-            get
-            {
-                return new TestLoggerFactory(Sink, true).CreateLogger<ResourceManagerStringLocalizer>();
-            }
-        }
+        private ILogger Logger => new TestLoggerFactory(Sink, enabled: true).CreateLogger<ResourceManagerStringLocalizer>();
```
Comment (hishamco) [inline test/Microsoft.Extensions.Localization.Tests/ResourceManagerStringLocalizerTest.cs]: 👍

**Comment by hishamco** [comment]
CI are failed!!

**Pull request merged**